docker-compose-sse.yaml

```diff
@@ -0,0 +1,74 @@
+version: '3.4'
+
+services:
+
+  nginx:
+    image: nginx
+    restart: on-failure
+    build: nginx
+    ports:
+      - "8080:8080"
+      - "8443:8443"
+    environment:
+      HOSTNAMES_DIR: "/hostnames"
+    volumes:
+      - "./tests/hostnames:/hostnames"
+    depends_on:
+      - minio
+
+  minio:
+    image: minio
+    restart: on-failure
+    build: .
+    environment:
+      MINIO_ROOT_USER: "12345678"
+      MINIO_ROOT_PASSWORD: "12345678"
+      MINIO_AUDIT_WEBHOOK_ENABLE_target1: "on"
+      MINIO_AUDIT_WEBHOOK_AUTH_TOKEN_target1: ""
+      MINIO_AUDIT_WEBHOOK_ENDPOINT_target1: http://cwm-worker-logger:8500/logs
+      MINIO_GATEWAY_DEPLOYMENT_ID: "docker-compose-http"
+      MINIO_KMS_AUTO_ENCRYPTION: "on"
+      MINIO_KMS_KES_ENDPOINT: "https://minio-kes:7373"
+      MINIO_KMS_KES_CERT_FILE: "/minio-kes/minio.cert"
+      MINIO_KMS_KES_KEY_FILE: "/minio-kes/minio.key"
+      MINIO_KMS_KES_CAPATH: "/minio-kes/server.cert"
+      MINIO_KMS_KES_KEY_NAME: "minio" # configured in `minio-kes/server-config.yaml`
+    volumes:
+      - "storage:/storage"
+      - "cache:/cache"
+      - "minio-kes:/minio-kes"
+    depends_on:
+      - minio-kes
+      - cwm-worker-logger
+
+  minio-kes:
+    image: minio-kes:v0.17.6
+    build: ./minio-kes
```

Comment:
are you using the default minio-kes image or building a custom image? this configuration is conflicting, if  you run docker-compose with --build it will build the custom image and tag it as `minio-kes:v0.17.6`, but if you don't build and pull the image before it will run with the pulled `minio-kes:v0.17.6` image - so two different images may be used depending on how you run it

Reply:
It's building the custom one i.e. `minio-kes`. The official one is `minio/kes`. Is it okay if the version is pinned here?

Reply:
best to remove the `image` property then, because it is meaningless for this purpose as we are not currently pushing this image to be available under that image name